packages/esm-patient-tests-app/src/config-schema.ts

```diff
@@ -48,6 +48,12 @@ export const configSchema = {
       _default: ['1748a953-d12e-4be1-914c-f6b096c6cdef'],
     },
   },
+  showLabReferenceNumberInput: {
```

Comment:
```suggestion
  showLabReferenceNumberField: {
```

nit: Feels to me like "Field" is arguably clearer than "Input" here